qiskit_ibm_runtime/fake_runtime_job.py

```diff
@@ -0,0 +1,80 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Qiskit fake runtime job."""
+
+from typing import Optional, Dict, Any
+
+from qiskit.primitives.primitive_job import PrimitiveJob
+from qiskit.providers import JobStatus, JobV1
+from qiskit.providers.fake_provider import FakeBackendV2 as FakeBackend
+
+# pylint: disable=cyclic-import
+# from .qiskit_runtime_service import QiskitRuntimeService
+
+
+class FakeRuntimeJob(JobV1):
+    """Representation of a runtime program execution on a simulator."""
+
+    def __init__(
+        self,
+        primitive_job: PrimitiveJob,
+        backend: FakeBackend,
+        job_id: str,
+        program_id: str,
+        # service: "qiskit_runtime_service.QiskitRuntimeService",
+        params: Optional[Dict] = None,
+        # creation_date: Optional[str] = None,
+        # user_callback: Optional[Callable] = None,
+        # result_decoder: Optional[Union[Type[ResultDecoder], Sequence[Type[ResultDecoder]]]] = None,
+        # tags: Optional[List] = None,
+    ) -> None:
+        """FakeRuntimeJob constructor."""
+        super().__init__(backend=backend, job_id=job_id)
+        # self._service = service  # do we need this?
```

Comment:
I added `creation_date` and `tags`. 
Removed `user_callback` as you wrote below and also because it refers to interim results. 
Removed `result_decoder` as you wrote below, because results will not be returned encoded.